docker-compose.yml

```diff
@@ -134,9 +147,8 @@ services:
       VRF_PUB: /kt/genfiles/vrf-pubkey.pem
       SIGN_KEY: /trillian/log-rpc-server.privkey.pem
       SIGN_KEY_PW: towel
+      SIGN_PERIOD: 5s
```

Comment:
The corresponding flag got renamed to `--min-period` for the signer. I would suggest to name this `MIN_SIGN_PERIOD` (Already fixed in #607: https://github.com/Liamsi/keytransparency/blob/6d3dde3ce28615df44bb80ac2076307c6e572362/cmd/keytransparency-signer/Dockerfile#L26). 
If this gets merged before #607, please update the flag in the Dockerfile (otherwise docker-compose won't succeed starting the kt-signer).